drivers/medicaid_hmis_interchange/app/jobs/medicaid_hmis_interchange/medicaid_id_lookup_job.rb

```diff
@@ -0,0 +1,45 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+require 'net/sftp'
+
+module MedicaidHmisInterchange
+  class MedicaidIdLookupJob < ::BaseJob
+    include NotifierConfig
+
+    def perform(client_ids, test: false)
+      soap = ::Health::Soap::MassHealth.new(test: test)
+      return unless soap.configured?
```

Comment:
This needs another guard -- otherwise, it will end up getting run in BHCHP.

Reply:
@awisema are you adding the guard now?  I just pushed a possible fix for cron syntax generation (guess we didn't have any weekly yet.)